control-station/src/services/PodSocketClient.ts

```diff
@@ -30,11 +44,17 @@ class PodSocketClient {
 	setPodData: SetPodData;
 
 	constructor(setPodData: SetPodData) {
-		this.socket = ioNamespace("control-station");
+		this.socket = ioNamespace("");
```

Comment:
Chore: let's keep the previously established `control-station` namespace

Reply:
Note from Rye: cannot use namespace right now since emitting from primary `io` reference rather than the specific `socket` reference defined. Will revisit.